pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -63,19 +63,19 @@
     from pennylane.operation import Tensor
     from pennylane.ops.op_math import Adjoint
     from pennylane.wires import Wires
+    from pennylane_lightning.lightning_kokkos_ops.algorithms import (
```

Comment:
Fun fact: that's `make format` that made this change. It doesn't seem to consider `pylint: ungrouped-imports` below.

Reply:
Yes, we should put a `pylint: disable=` right on top if `from pennylane_lightning.lightning_kokkos_ops.algorithms import` to fix this.

Reply:
I already tried this locally. The `make format` still moves the code snippet about `algorithms` to the top. But I will commit now, may be it resolves the CodeFactor issue.

Reply:
CodeFactor is happy